waggle-dance-integration-tests/src/test/java/com/hotels/bdp/waggledance/metrics/MonitoringConfigurationIntegrationTest.java

```diff
@@ -1,72 +0,0 @@
-/**
```

Comment:
How are we going to test this stuff?

Reply:
+1 would be good if we can still do this.
I haven't looked at micrometer enough, but I assume spring lifecycle opens/closes everything for us now and we don't have to do that ourselves like we did before? 
@abhimanyugupta07 I can help you with this test if you like

Reply:
Yeah. Let me see if it can still be done. I will ping you for this if needed.